benchmarking/commons/utils.py

```diff
@@ -15,6 +15,11 @@
 import numpy as np
 
 from syne_tune.util import s3_experiment_path
+from syne_tune.constants import (
+    ST_METADATA_FILENAME,
```

Comment:
This is cleaning things up. In the old code, file names were hardcoded. Now, they are in ``constants.py``